.travis.yml

```diff
@@ -0,0 +1,24 @@
+sudo: required
+dist: trusty
+language: generic
+
+cache:
+  directories:
+    - $HOME/.ccache
+
+env:
+  global:
+    - CCACHE_DIR=$HOME/.ccache
+
+  matrix:
+    - ROS_DISTRO="kinetic" ABICHECK_URL='github:ros-visualization/rviz#kinetic-devel' ABICHECK_MERGE=auto
+
+matrix:
+  allow_failures:
+    - env: ROS_DISTRO="kinetic" ABICHECK_URL='github:ros-visualization/rviz#kinetic-devel' ABICHECK_MERGE=auto
```

Comment:
Note that by `allow_failures`-ing, you may not easily notice on GitHub PR view even if the failure occurred  (sorry for being pedantic).

Reply:
Happy to accept input, thanks! Sounds like it's best to remove it for increased visibility in the event that it doesn't pass.